Packs/JWT/Integrations/JWT/JWT.yml

```
@@ -0,0 +1,243 @@
+category: Authentication
+commonfields:
+  id: JWT
+  version: -1
+configuration:
+- additionalinfo: The authentication server URL
+  defaultvalue: https://example.com/
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- additionalinfo: The secret key to sign the authentication token.
+  display: Secret Key
+  hidden: false
+  name: key
+  required: true
+  type: 0
```

```suggestion
  type: 4
```
sensitive parameters should be encrypted

---

noted, will fix that